cmd/project.go

```diff
@@ -72,7 +72,17 @@ func (p *Project) createLicenseFile() error {
 	data := map[string]interface{}{
 		"copyright": copyrightLine(),
 	}
-	licenseFile, err := os.Create(fmt.Sprintf("%s/LICENSE", p.AbsolutePath))
+
+	licensePath := p.AbsolutePath + "/LICENSE"
+
+	if p.Legal.Name == "None" {
+		// If user didn't set any license and an existing license file is found, use that.
+		if _, err := os.Stat(licensePath); err == nil {
+			return nil
+		}
+	}
+
+	licenseFile, err := os.Create(licensePath)
```

Comment:
Using exiting license makes sense, but this check ignores errors from os.Stat(), (other then fs.ErrNotExits). If the file exits and os.Stat failed, something is very wrong and failing is the best option.

But I think there is a better way - do we really want to overwrite existing license file if the user specify the license type? This does not feel like a careful way to modify user source tree.

So we can ensure that we never overwrite existing license, and handle the special case when the user did not specify the license, in which we will reuse the existing license.

Something like:

```go
licenseFile, err := os.OpenFile(licensePath, O_CREATE|O_EXCL, 0755)
if err != nil {
    if errors.Is(err, fs.ErrExist) && p.Legal.Name == "None" {
        // User did not specify the license - use the existing one.
        return nil
    }
    return err
}
```